app/src/main/java/com/beemdevelopment/aegis/ui/AuthActivity.java

```diff
@@ -83,6 +86,10 @@ protected void onCreate(Bundle savedInstanceState) {
             return false;
         });
 
+        if (_prefs.isPinKeyboardEnabled()) {
+            _textPassword.setInputType(InputType.TYPE_CLASS_NUMBER);
```

Comment:
After this call, the text in the password field is no longer hidden, and the password toggle also disappears. I think we need to combine two flags: 

```
_textPassword.setInputType(InputType.TYPE_CLASS_NUMBER | InputType.TYPE_NUMBER_VARIATION_PASSWORD);
```

The same goes for the call to change the type back to password.